### DISCUSSION TITLE
Hot reload not working if tailwind directives included not in src/styles.scss

### DISCUSSION THREAD

**Discussion by pleshakovalexander** [tailwindlabs/tailwindcss#11112] [Help]
I have an angular app that exposes as remoteEntry for module federation app so i can't include tailwind in root app styles. 

I have a component in app folder and in that component styles file i added the Tailwind directives

 **src/app/my-component/my-component.scss**
```
@tailwind base;
@tailwind components;
@tailwind utilities;
```

when i run ng build all tailwind classes builded correctly, but then i add new classes to my templates tailwind will not rebuild with new classes.

if i import Tailwind directives into **/src/styles.css** all works as expected. But default solution not suited for my app because this angular app is used in module federation app and main app don't use tailwind

**Comment by pleshakovalexander** [comment]
Please any help

**Comment by asmin52** [comment]
This works really well for me, thank you! Facing same issue here. Help is appreciated. 
[Price Chopper Direct Connect Login](https://www.pricechopperdirectconnect.org/)
